assets/js/load_more.js

```diff
@@ -0,0 +1,36 @@
+jQuery(document).ready(function () {
+  initLoadMore();
+})
+
+function initLoadMore() {
+  jQuery('.jtf-load-more').on('click', function (e) {
+    e.preventDefault();
+    var $loadmore = jQuery(this),
+      link = $loadmore.attr('href'),
+      selector = $loadmore.attr('data-selector'),
+      container = $loadmore.attr('data-container');
```

Comment:
you can use this construction:

`container = $loadmore.attr('data-container') || selector;`